docs/sdk/io/runtask.mdx

```diff
@@ -201,4 +219,43 @@ client.defineJob({
 });
 ```
 
+```typescript Remote callbacks
+client.defineJob({
+  id: "remote-callback-example",
+  name: "Remote Callback example",
+  version: "0.1.1",
+  trigger: eventTrigger({ name: "predict" }),
+  integrations: { replicate },
+  run: async (payload, io, ctx) => {
+    //runTask
+    const prediction = await io.runTask(
```

Comment:
I think this is meant to be `io.replicate.runTask` right?

Reply:
Yes, good catch!

Reply:
Had to update the GitHub example too, as underlying clients are not exposed at `io.integration.client` anymore. Also added a section to mention the wrappers at `io.integration.runTask()`.